src/AccordionItemTitle/accordion-item-title.js

```diff
@@ -48,6 +48,22 @@ class AccordionItemTitle extends Component {
             },
         );
 
+        if (role === 'tab') {
+            return (
+                <div // eslint-disable-line jsx-a11y/no-static-element-interactions
```

Comment:
This linting rule is just getting in the way, you should disable it for the whole project.